### PR TITLE
fix: avoid lowercase criteria filter

### PR DESCRIPTION
**Pull request by gabrielfs7**
We should not lowercase/transform criteria in order to have it working in the search.

https://oat-sa.atlassian.net/browse/ADF-666

### CODE REVIEW

**Review by chinnu-focaloid — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by boajer — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: test/ElasticSearchTest.php [hunk 1]
```diff
@@ -149,20 +149,7 @@ public function testSearch(): void
 
         $this->assertEquals(new SearchResult([], 0), $this->sut->search($query));
     }
-
-    public function testQuery_callGenerisSearchCaseClassIsNotSupported(): void
```
Comment (gabrielfs7) [inline test/ElasticSearchTest.php:153]: Test removed, cause this validation does not exist anymore

**Comment by AntonNeviarouski** [comment]
QA verified

**Pull request merged**